packages/preview/simple-preavis/0.1.0/typst.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "simple-preavis"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Mathias APARICIO <https://github.com/mathias-aparicio/>"]
+license = "MIT"
+description = "📖 a french move out letter"
+repository = "https://github.com/mathias-aparicio/simple-preavis"
+categories = ["letter"]
+exclude = [".gitignore"]
+
+[template]
+entrypoint="lib.typ"
+path="."
+thumbnail = "thumbnail.png"
```

This won't work as expected: I think you should create a `template` subfolder in which you move your `example.typ` file, and then update the manifest to the following:

```suggestion
[template]
entrypoint = "example.typ"
path = "template"
thumbnail = "thumbnail.png"
```

With your current code, the whole package would be copied when creating a project from this template.